release-notes.md

```diff
@@ -1,9 +1,17 @@
 # Release notes
 
-## October 3, 2019
+## October 4, 2019
 
 The following updates were made in this release:
 
+**iOS Core 2.3.4**
+* Fixed a crash that may happend during app shutdown.
```

Comment:
Please reword to:
"Fixed a crash that might have happened during app shutdown."